packages/wallet-sdk/src/sign/scw/message/type/Action.ts

```diff
@@ -127,6 +131,36 @@ export type AddEthereumChainAction = {
   };
 };
 
+export type GetCapacitiesAction = {
+  method: SupportedEthereumMethods.WalletGetCapacities;
+  params: null;
+};
```

Comment:
should be called capacities --> capabilities. also takes a single argument: a chain id. capabilities could differ per chain so method will take a chain id and return capabilities for that chain id. also think it should be hex, chain ids are uint256

Reply:
I think this would mean that for every chain switch, the popup would need to open briefly :/ . Not ideal.

Reply:
ok per discussion elsewhere we'll receive and store capabilities per chain during the handshake - updates here [2e350be](https://github.com/coinbase/coinbase-wallet-sdk/pull/1138/commits/2e350befe010b3902d7353498a2b5b4baa6e6f13)